bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -806,6 +824,11 @@ types to vary over time.
 (Only relevant if FATES is on).
 </entry>
 
+<entry id="flandusepftdat" type="char*256" category="datasets"
+       input_pathname="abs" group="clm_inparm" valid_values="" >
+Full pathname of fates landuse x pft static data map.
```

Comment:
Mention the logical flag that's makes this required.

Reply:
Addressed via 387ac5e